scan/lib/scan/test_command_generator.rb

```diff
@@ -37,6 +37,7 @@ def options
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
         options << "-enableCodeCoverage YES" if config[:code_coverage]
         options << "-enableAddressSanitizer YES" if config[:address_sanitizer]
```

Comment:
I would say that as a rule all options should reflect the user's explicit configuration. This should be a rule for all tools/wrappers.